antarest/study/storage/variantstudy/variant_command_generator.py

```diff
@@ -89,8 +102,9 @@ def _generate(
             }
             results.details.append(detail)
 
-            if notifier:
-                notifier(index - 1, output.status, output.message)
+            if notifier and cmd.command_id not in command_block_set:
+                notifier(command_block_dict[cmd.command_id], output.status, output.message)
```

Comment:
@maugde @MartinBelthle : 
Actually it's weird to notify only on the first command of a block.
In particular, we will notify with the status of this first command, but not for subsequent commands:
so for example if the first command succeeds and the second one fails, it will be invisible from the listener.

In the end I would propose to change the scope of this work:
I think the notification mechanism is actually not used (it just sends an event to the event bus, but this event type is never used for now. I cannot see a usage in the near future.

--> I propose to simply delete this notifier and the associated event type